.agents/scripts/report_render_styles.py

```
@@ -223,6 +223,22 @@ def _flatten_token_mapping(value: object, prefix: str = "") -> dict[str, str]:
+def _parse_section_header(stripped: str, indent: int) -> tuple[str, str] | None:
+    if indent == 0 and stripped.endswith(":"):
+        return stripped[:-1], ""
+    return None
+
+
+def _parse_section_token(section: str, nested: str, stripped: str, indent: int) -> tuple[str, tuple[str, str] | None]:
+    prefix_by_section = {"colors": "", "rounded": "rounded."}
+    if section in prefix_by_section and indent == 2:
+        return nested, _parse_mapping(stripped, prefix_by_section[section])
+    if section == "typography":
+        next_nested, key, value = _parse_typography_line(stripped, nested, indent)
```

Ruff (0.15.13)

[error] Undefined name `_parse_typography_line`

(F821)